modules/functional_expansion_tools/src/series/Zernike.C

```diff
@@ -202,6 +229,25 @@ Zernike::evaluateOrthonormal()
 }
 // clang-format on
 
+void
+Zernike::evaluateSqrtMu()
+{
+  evaluateStandard();
+  save(0, load(0) / std::sqrt(M_PI));
+  size_t j = 1;
+  for (size_t n = 1; n < _orders[0] + 1; ++n)
+  {
+    for (size_t m = 0; m < n + 1; ++m)
+    {
+      if (m != 0 && n / m == 2 && n % m == 0)
+        save(j, load(j) * std::sqrt((2 * n + 2) / (2 * M_PI)));
```

Comment:
Same thing here: `(2 * n + 2) / (2 * M_PI)` = `(n + 1) / M_PI`

Again, just a minor comment. Perhaps a compiler optimization will reduce this anyway.